src/actions/remove.js

```diff
@@ -30,24 +33,27 @@ import { extendMetaWithResponse, buildRSAAConfig } from '../rsaa';
  * @returns {{}}
  */
 export function remove(schema, item, params = {}, options = {}) {
-  const config = resolveConfig(schema);
+  const config = resolveResourceConfig(schema, 'remove');
   if (!config) {
-    const schemaName = schema && _.isObject(schema) ? schema.schema : schema;
+    const schemaName = schema && _.isObject(schema) ? resolveSchemaType(config) : schema;
     throw new Error(`Couldn't resolve schema ${schemaName} in function find.`);
   }
   if (!_.isObject(item)) {
     throw new Error('Item isn\'t object.');
   }
 
+  const normalizedItem = normalize(item, resolveSchema(config));
+
   const rsaaConfig = buildRSAAConfig(config);
   const endpoint = buildEndpoint(rsaaConfig.endpoint, params, options);
+  const source = _.get(config, 'request.resourceType', JSON_API_SOURCE);
 
   const meta = {
-    source: config.request.resourceType || JSON_API_SOURCE,
-    schema: config.schema,
     params,
     endpoint,
     options,
+    source,
+    schema: resolveSchemaType(config),
```

Comment:
You are calling `resolveSchemaType` multiple times here. Is this necessary? Could you resolve it once at the top of this function?